Tryouts/Prototypes/MorganStanley.ComposeUI.Prototypes.NoPluginsPrototype/App.axaml.cs

```diff
@@ -0,0 +1,50 @@
+using Avalonia;
```

Comment:
License header is missing. BTW we have the license header in the solution level `.dotsettings` file, you can use ReSharper to update it in all `.cs` files.

Reply:
done